rules/renaming/src/ValueObject/RenameStaticMethod.php

```diff
@@ -46,7 +46,7 @@ public function getOldMethod(): string
 
     public function getNewClass(): string
     {
-        return $this->newClass;
+        return '\\' . $this->newClass;
```

Comment:
I think this should be fixed with `FullyQualified` in the place it's used

Reply:
implemented.